jekyll/_cci2/build-processing.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: classic-docs
+title: "Enabling Build Processing"
+short-title: "Enabling Build Processing"
+description: "How to enable the preview of build processing"
+categories: [settings]
+order: 1
+---
+
+# CircleCI Build Processing
+
+This document describes how to enable the preview build processing engine if you need to trigger workflows from the CircleCI API.
```

Comment:
May want to include more information here around build processing also enabling version 2.1 for config, unless perhaps you are keeping that out until we come out of Preview? Is the intention we'll link here instead of to the Preview SDK repo?

Reply:
Yep, I intentionally left 2.1 out of this for now and would prefer to move those docs over and link to them when it is supported